docs/notebooks/analysis/swept_langmuir/find_floating_potential.ipynb

```diff
@@ -0,0 +1,273 @@
+{
```

Comment:
<p>resulst -&gt; results</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/889/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
I always seem to mix up those last three letters.  I can't tell you how many times I've corrected that. Arg!